content/en/building/tasks/tasks-overview.md

```diff
@@ -32,6 +32,35 @@ Each task has an icon on the left side which indicates which type of task it is.
 Tasks are listed in order of due date. Tasks that are past due will appear at the top of the list. CHWs should strive to complete tasks before they are overdue. Many programs add targets to track task completion and timeliness.
 
 
+<br clear="all">
+
+## Task prioritization
+
+{{< callout >}}
+Available from 4.21.0.
+{{< /callout >}}
+
+Within the community-based health workflow, where volumes are large, Community Health Volunteers (CHVs) are commonly faced with dozens of active tasks. Tasks are sorted by due date by default, which can make it hard to prioritise the most important or urgent action, particularly when tasks have similar due dates or where task urgency is led by clinical factors, rather than scheduling considerations alone.
+
+{{< cards rows="4" >}}
+{{< card link="tasks-prioritization.png" image="tasks-prioritization.png"  method="resize">}}
+{{< /cards >}}
+
+To assist with task management and decision-making in the field, the CHT supports custom task prioritization logic. Task prioritization seeks to:
+- Assist CHVs in prioritizing activities that have the greatest impacts without having to review them manually.
+- Factor in signs of clinical or operational urgency, such as danger signs.
+- Facilitate predictive and algorithm-based sorting guided by future risk or predicted behavior.
+- Enhance filtering to see the most important or urgent tasks only.
```

Comment:
Can you please add some detail into what prioritization actually does? You are mentioning that tasks are sorted by due date by default, but do not explain what this new feature does.